app/controllers/alaveteli_pro/subscriptions_controller.rb

```diff
@@ -87,18 +87,11 @@ def show
   private
 
   def authenticate
-    case action_name
-    when 'create'
-      post_redirect_params = {
-        :web => _('To upgrade your account'),
-        :email => _('To upgrade your account'),
-        :email_subject => _('To upgrade your account') }
-    else
-      post_redirect_params = {
-        :web => _('To access your account'),
-        :email => _('To access your account'),
-        :email_subject => _('To access your account') }
-    end
+    post_redirect_params = {
+      :web => _('To upgrade your account'),
+      :email => _('Then you can upgrade your account'),
```

Comment:
Oh, it's always going to say 'upgrade' even if you're creating a new account 🤔

Reply:
I don't have a problem with that – it still conveys what's happening.